.github/workflows/release-process.yml

```diff
@@ -40,10 +40,18 @@ jobs:
           cd cli
           ./hack/build-binaries.sh
           shasum -a 256 ./kctrl-* | tee -a ../tmp/checksums.txt
-          echo "# :open_file_folder: Files Checksum" | tee ../tmp/checksums-formatted.txt
-          echo '```' | tee -a ../tmp/checksums-formatted.txt
-          cat ../tmp/checksums.txt | tee -a ../tmp/checksums-formatted.txt
-          echo '```' | tee -a ../tmp/checksums-formatted.txt
+
+      - name: Run Package build
+        run: |
+          constraintVersion="${{ github.ref_name }}"
+          ./cli/kctrl-linux-amd64 pkg release -y -v ${constraintVersion:1} --debug
+          mv ./carvel-artifacts/packages/kapp-controller.carvel.dev/metadata.yml ./carvel-artifacts/packages/kapp-controller.carvel.dev/package-metadata.yml
```

Comment:
nit: I think we can `cd` to `carvel-artifacts/packages/kapp-controller.carvel.dev` as next three lines relate to this directory and these paths are long.

Reply:
```
++ cat
7,8c7,8
< 21dc59afe62c27513a177069abd0cf9503ae2fb3f58255ba611b7f7f1faaae33  ./package.yml
< 780be38c25a4044772f463502eb8267333784168de01d8ca146ecaa9c0d87cbe  ./package-metadata.yml
---
> 21dc59afe62c27513a177069abd0cf9503ae2fb3f58255ba611b7f7f1faaae33  ./carvel-artifacts/packages/kapp-controller.carvel.dev/package.yml
> 780be38c25a4044772f463502eb8267333784168de01d8ca146ecaa9c0d87cbe  ./carvel-artifacts/packages/kapp-controller.carvel.dev/package-metadata.yml
Error: Process completed with exit code 1.
```
In doing this verifying the uploaded artefacts stage get fails as it is also matching the path and can't do `cd` in `Create release draft and upload release yaml` step.

Reply:
and also we have to make changes in path from `tee -a ./tmp/checksums.txt` to `tee -a ./../../../tmp/checksums.txt` everywhere